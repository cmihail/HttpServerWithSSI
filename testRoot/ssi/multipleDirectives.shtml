<html>
<head>
	<title>SSI Multiple Directives</title>
</head>

<body>
	<!--#include file="included.html" -->
	<!--# invalidDirective file="included.html"-->
	Random text
	<!--#exec cmd="ls execContent" -->
	<!--#config timefmt="%A" time="%B" --> <!--#echo   var="DATE_LOCAL" -->
	<!--#include file="included.html" ignored="other" -->
</body>
</html>
