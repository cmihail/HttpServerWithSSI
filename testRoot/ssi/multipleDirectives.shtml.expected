<html>
<head>
	<title>SSI Multiple Directives</title>
</head>

<body>
	<b><i>Test</b></i>
<br />
This is a test ....
	<!--# invalidDirective file="included.html"-->
	Random text
	execCmd.shtml
execCmd.shtml.expected

	 
	<b><i>Test</b></i>
<br />
This is a test ....
</body>
</html>
